chapters/08-erros-e-manipulacao-de-erros.md

```diff
@@ -6,11 +6,11 @@
 > Yuan-Ma havia escrito um pequeno programa onde utilizou muitas variáveis globais e atalhos que faziam a qualidade do seu código inferior. Lendo o programa, um estudante perguntou: “Você nos avisou para não usar essas técnicas e mesmo assim as encontro no seu programa. Como pode isso?”. O mestre respondeu: “Não há necessidade de se buscar uma mangueira de água quando a casa não está em chamas.”
 > — Master Yuan-Ma, The Book of Programming
 
-Um programa são pensamentos cristalizados. Às vezes esses pensamentos são confusos. Outras vezes os erros são introduzidos ao converter esse pensamento em código. De qualquer forma o resultado será um programa falho.
+Um programa é um pensamento “cristalizado”. Algumas vezes, esses pensamentos são confusos e erros podem ser inseridos quando convertemos pensamentos em código, resultando em um programa com falhas.
 
-Falhas em um programa são considerados como bugs. Bugs podem ser erros causados pelo programador ou problemas em outros sistemas na qual o programa interage. Alguns erros podem serem detectados facilmente enquanto outros são extremamente sutis e pode permanecer escondido em um sistema por anos.
+Falhas em um programa são normalmente chamadas de “bugs”, e podem ser causadas por erros inseridos pelo programador ou problemas em outros sistemas que a aplicação interage. Alguns *bugs* são imediatamente aparentes, enquanto outros são sutis e podem ficar escondidos em um sistema por anos.
```

Comment:
Eu gosto bastante da ideia de usarmos itálico para todas as ocorrências de termos em inglês.

Sobre as aspas você está 100% correto. Se usarmos uma vez, temos que usar em todas as ocorrências. Eu fiz uma consulta com uma pessoa que transcreve/edita textos profissionalmente e esses foram os pontos:
* As aspas são opcionais nos casos de figura de linguagem (sentido figurado)
* Quando fazemos comparações  `[...] são **como** pequenos universos` não tem necessidade de aspas
* As aspas nesses casos são usadas para dar destaque e ênfase
* Não usar aspas em algumas palavras que tem um sentido claro para o leitor (ex: corpo de uma função) faz com que o texto fique mais limpo

Por isso eu votaria em:
1. Convencionar que **todos** os termos em inglês sejam em *itálico*
2. Evitar ao máximo usar as aspas.  Usar somente nos casos em que queremos dar destaque/ênfase mesmo em algum termo bem específico que seja usado de forma figurada (lembrando que comparações não entram).